ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/services/collectors/ModifierExtractor.java

```diff
@@ -0,0 +1,54 @@
+package edu.usb.argos.ASTProcessor.visitor.core.services.collectors;
+
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.ModifierType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class ModifierExtractor {
+    public List<String> extractModifiers(JavaParser.ClassBodyDeclarationContext ctx) {
+        return Optional.ofNullable(ctx)
+                .map(JavaParser.ClassBodyDeclarationContext::modifier)
+                .map(this::processModifierList)
+                .orElse(Collections.emptyList());
+    }
+
+    private List<String> processModifierList(List<JavaParser.ModifierContext> modifierContexts) {
+        List<String> modifiers = new ArrayList<>();
+
+        for (JavaParser.ModifierContext mod : modifierContexts) {
+            processModifier(mod, modifiers);
+        }
+
+        return modifiers;
+    }
+
+    private void processModifier(JavaParser.ModifierContext mod, List<String> modifiers) {
+        JavaParser.ClassOrInterfaceModifierContext classOrInterfaceModifier = mod.classOrInterfaceModifier();
+        if (classOrInterfaceModifier == null) {
+            return;
```

Comment:
Instead of putting a return; it would be good if you did not put the method as void but instead returned something so that you do not have to put return; or do you think it is necessary this way?